docs/pages/guides/modes.mdx

```diff
@@ -0,0 +1,82 @@
+import { Markdown } from '../../components/Page';
+
+# Modes
+
+With the support of SQLite in Keystone 6, you can now choose to use Keystone in a “standalone” (headless) fashion, or “embed” your backend within a frontend app.
+
+We refer to these two use cases as “modes”. **Embedded mode is new**, and offers a different approach to integrating Keystone. While it offers some quick wins (for developers in particular), it also comes with restrictions that constrain it to a narrow set of uses at this point in time. 
+
+!> Modes is conceptual framework. It has no dedicated APIs, and there are no mentions of modes in the API docs.
+
+## Standalone mode
+
+Standalone mode is the default (headless) way to use Keystone. In fact **prior to version 6, standalone mode was the only way you could use Keystone**. The premise of standalone mode is:
```

Comment:
I know I called it out above but I'm OK with the `version 6` here, this sentence is going to be awkward otherwise, so lets leave this one

Reply:
Actually, I could say `from versions 5 and lower, standalone mode was the only way you could use Keystone`

Reply:
I did the above ^